dawn/src/dawn/SIR/VerticalRegion.cpp

```diff
@@ -0,0 +1,181 @@
+#include "dawn/SIR/AST.h"
+#include "dawn/SIR/ASTStringifier.h"
+#include "dawn/SIR/ASTVisitor.h"
+#include "dawn/SIR/VerticalRegion.h"
+#include "dawn/Support/Casting.h"
+#include "dawn/Support/Format.h"
+#include "dawn/Support/StringUtil.h"
+#include <sstream>
+
+namespace dawn {
+
+CompareResult sir::VerticalRegion::comparison(const sir::VerticalRegion& rhs) const {
```

Comment:
I suggest to move it below where you open a `namespace sir`

Reply:
Done.